pkg/api/http.go

```diff
@@ -23,13 +23,14 @@ import (
 const (
 	URLParamTraceID = "traceID"
 	// search
-	urlParamQuery       = "q"
-	urlParamTags        = "tags"
-	urlParamMinDuration = "minDuration"
-	urlParamMaxDuration = "maxDuration"
-	urlParamLimit       = "limit"
-	urlParamStart       = "start"
-	urlParamEnd         = "end"
+	urlParamQuery           = "q"
+	urlParamTags            = "tags"
+	urlParamMinDuration     = "minDuration"
+	urlParamMaxDuration     = "maxDuration"
+	urlParamLimit           = "limit"
+	urlParamStart           = "start"
+	urlParamEnd             = "end"
+	urlParamSpansPerSpanSet = "spansPerSpanSet"
```

Comment:
Tempo can make 10s or even 100s of thousands of jobs for a given query. Is there a shorter parameter name we can use that will make sense to reduce bandwidth between components for distributing jobs?

Reply:
spss?

Reply:
works for me